R/workflow.R

```diff
@@ -395,22 +405,18 @@ copy_nonmem_model_from <- function(
   .new_mod[[YAML_MOD_PATH]] <- basename(new_mod_path) # path should be relative to YAML location
 
   # copy control steam to new path
-  if (.update_mod_file) {
-    parent_mod_id <- .parent_mod %>% get_model_path() %>% get_model_id()
-    new_mod_id <- .new_model %>% get_model_id()
-
-    # read parent control stream
-    mod_str <- .parent_mod %>% get_model_path() %>% read_file()
-
-    # replace the $PROBLEM line(s)
-    mod_str <- str_replace(mod_str,
-                "\\$PROB(.|\n)*?\\$",
-                as.character(glue("$PROBLEM {new_mod_id} {.description}\n\n$")))
-
-    # read parent control stream
-    write_file(mod_str, get_model_path(.new_mod))
+  .new_model_path <- .new_mod %>% get_model_path()
+  .parent_model_path <- .parent_mod %>% get_model_path()
+
+  if (fs::file_exists(.new_model_path)) {
+    if (isTRUE(.overwrite_model_file)) {
+      warning(glue("Overwriting file at {.new_model_path} because `.overwrite_model_file=TRUE`"))
```

Comment:
given overwrite= TRUE is a user forced setting, I would not message/warn by default here since the only way to get through this path is if the user explicitly sets it